packages/server/lib/cloud/api.ts

```diff
@@ -11,7 +11,11 @@ const pkg = require('@packages/root')
 
 const machineId = require('./machine_id')
 const errors = require('../errors')
-const { apiRoutes } = require('./routes')
+const { apiUrl, apiRoutes, makeRoutes } = require('./routes')
+
+import Bluebird from 'bluebird'
```

Comment:
`bluebird` is already required on line 5.

Reply:
Ah I see required as `Promise` - gonna update all `Promise` references in the file to be `Bluebird` so it's more explicit